common/app/model/content.scala

```diff
@@ -552,7 +552,7 @@ object Video {
 
 final case class Video (
   override val content: Content,
-  source: Option[String] ) extends ContentType {
+  source: Option[String] ) extends ContentType with Page {
```

Comment:
Don't think we want to combine the concept of a `ContentType` with a `Page`, because we shouldn't always assume a capi item is being used in the context of a page. It could be a trail, for example, or a related item, etc.

See above...